From 042ecdc2e6f8aa10e16af534fa0f1485c0a6234b Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Thu, 18 Jan 2024 08:03:50 +0200
Subject: [PATCH] [FSW-402] gaudi2: add precompiled binaries

This patch adds precompiled binaries for gaudi2 arcboot
build. This will allow complete restructuing of arcboot code
without dependency on gaudi platforms.

Change-Id: Ifa322eb25722e9d816aa659693c13223d792ca65
---

diff --git a/bin/gaudi2_arc/asic/arcboot_rom.bin b/bin/gaudi2_arc/asic/arcboot_rom.bin
new file mode 100644
index 0000000..707165c
--- /dev/null
+++ b/bin/gaudi2_arc/asic/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi2_arc/asic/arcboot_rom.elf b/bin/gaudi2_arc/asic/arcboot_rom.elf
new file mode 100644
index 0000000..f51d261
--- /dev/null
+++ b/bin/gaudi2_arc/asic/arcboot_rom.elf
Binary files differ
diff --git a/bin/gaudi2_arc/onix/arcboot_rom.bin b/bin/gaudi2_arc/onix/arcboot_rom.bin
new file mode 100644
index 0000000..707165c
--- /dev/null
+++ b/bin/gaudi2_arc/onix/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi2_arc/onix/arcboot_rom.elf b/bin/gaudi2_arc/onix/arcboot_rom.elf
new file mode 100644
index 0000000..f51d261
--- /dev/null
+++ b/bin/gaudi2_arc/onix/arcboot_rom.elf
Binary files differ
diff --git a/bin/gaudi2_arc/pldm/arcboot_rom.bin b/bin/gaudi2_arc/pldm/arcboot_rom.bin
new file mode 100644
index 0000000..707165c
--- /dev/null
+++ b/bin/gaudi2_arc/pldm/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi2_arc/pldm/arcboot_rom.elf b/bin/gaudi2_arc/pldm/arcboot_rom.elf
new file mode 100644
index 0000000..f51d261
--- /dev/null
+++ b/bin/gaudi2_arc/pldm/arcboot_rom.elf
Binary files differ
